modules/nf-core/lofreq/alnqual/main.nf

```diff
@@ -0,0 +1,51 @@
+process LOFREQ_ALNQUAL {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/lofreq:2.1.5--py310h47ef89e_10' :
+        'biocontainers/lofreq:2.1.5--py310h47ef89e_10' }"
+
+    input:
+    tuple val(meta), path(bam)
```

Comment:
Can you add a quick check to see if the input and output filename are different? See [here](https://github.com/nf-core/modules/blob/e809c6b078d5343bdf8b5b2b78483096a2b5a973/modules/nf-core/svync/main.nf#L25) for an example

Reply:
Added this.